adflow/pyADflow.py

```diff
@@ -3318,6 +3541,27 @@ def solveAdjoint(self, aeroProblem, objective, forcePoints=None,
                 self.curAP.adflowData.adjoints[objective] = psi
                 self.curAP.adjointFailed = False
 
+    def _process2_outer(self, dvName, TotalDeriv_):            
```

Comment:
these routines are called from the fwd mode total derivative stuff above so they should be checked together